themes/SuiteP/js/style.js

```diff
@@ -378,9 +378,10 @@ var isUserProfilePage = function() {
         module = $('#EditView_tabs').closest('form#EditView').find('input[name="module"]').val()
     }
     if(!module) {
-        module = module_sugar_grp1;
+        // ISSUE: breaks detailview and list views
```

Comment:
Can't seem to find an issue with this being uncommented.  Reinstate this back in please.